### PR TITLE
Upgrade to Python 3.11

### PR DESCRIPTION
**Pull request by lcduong**
> **Short description of what this resolves:** Upgrade to Python 3.11
> 
> This solves #154  
> 
> **Changes proposed in this pull request:**
> 
> * Upgrade to Python 3.11
> 
> **Checklist**
> 
> * [x]   I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
> * [x]   My branch is up-to-date with the Upstream development branch.
> * [ ]   The acceptance, integration, unit tests and linter pass locally with my changes
> * [ ]   I have added tests that prove my fix is effective or that my feature works
> * [ ]   I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by hongquan — APPROVED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -15,8 +15,7 @@ classifiers = [
   "Topic :: Internet :: WWW/HTTP :: Dynamic Content",
   "Environment :: Web Environment",
   "License :: OSI Approved :: Apache License, Version 2.0",
-  "Programming Language :: Python :: 3.8",
-  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.11",
   "Framework :: Django :: 3.2",
```
Comment (hongquan) [inline pyproject.toml]: Could you change this line, too. We are using Django 4.2.

**Review by mariobehling — APPROVED** [review]
Works when testing the deployment.

**Review by mariobehling — CHANGES_REQUESTED** [review]
Please still change  "Framework :: Django :: 3.2", -> 4.2

File: pyproject.toml (hunk 1 above)
Comment (norbusan) [reply pyproject.toml]: I made the changes directly in the github UI

**Pull request merged**